src/contract_client/utils.ts

```diff
@@ -1,7 +1,11 @@
+// @ts-ignore
+import * as jsxdr from "@stellar/js-xdr";
```

Comment:
Import only what you need!

```suggestion
import { XdrReader } from "@stellar/js-xdr";
```